eclipse-theia/theia
# navigator: fix hidden toolbar icons when resizing

#### What it does

This PR fixes #12602 by changing the styling of the explorer to make sure that the toolbar icons are always visible and that the folder name has an ellipsis overflow when the width of the `Explorer` is limited.

Demo:
![Demo](https://github.com/eclipse-theia/theia/assets/86936229/1602037b-4618-465c-a2ba-fab38ffe543f)

#### How to test


- Open a folder with a long name
- Resize the `Explorer` view, reducing its width
- Toolbar icons should be visible and the folder name should have an ellipsis overflow

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
